lib/plugins/layer2/strike/strike.js

```diff
@@ -1,5 +1,5 @@
 const axios = require('axios')
-const _ = require('lodash/fp')
+const _ = require('lodash')
```

Comment:
We always use `lodash/fp` as it allows us to be more functional. It's important to keep this consistent.